provision/ubuntu/install.sh

```diff
@@ -51,7 +51,7 @@ sudo apt-get install -y --allow-downgrades \
     protobuf-compiler libprotobuf-dev libyaml-cpp-dev \
     socat pv tmux bc gcc-multilib binutils-dev \
     binutils wget rsync ifupdown \
-    python-sphinx python-pip \
+    python3-sphinx python3-pip \
```

Comment:
I'm ok with the change, but I would not install here python3-sphinx and I would use the sphinx that you have in the line 74.